packages/p2p/src/components/advertiser-page/block-user/block-user-count.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { Icon, Popover, Text } from '@deriv/components';
+import { localize } from 'Components/i18next';
+import './block-user-count.scss';
+import { useStores } from 'Stores';
+
+const BlockUserCount = () => {
+    const { my_profile_store } = useStores();
+    const { blocked_by_count } = my_profile_store.advertiser_info;
+
+    React.useEffect(() => {
+        my_profile_store.getAdvertiserInfo();
+    }, []);
+
+    return (
+        <Popover
+            className='block-user-count'
+            classNameTarget='block-user-count__container'
+            alignment='top'
+            message={localize(
+                blocked_by_count === 0
+                    ? 'Nobody has blocked you. Yay!'
+                    : '{{blocked_by_count}} people have blocked you.',
+                { blocked_by_count }
```

Comment:
Instead of using a ternary within localize, put it outside and have two separate localize as there will be translations issues